app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -25,6 +27,14 @@ def index
         end
       end
 
+      # Returns all pages as a tree from the root given by the id parameter
+      #
+      def tree
+        authorize! :tree, :alchemy_admin_pages
+
+        render json: PageTreePresenter.new(@page, ability: current_ability, user: current_alchemy_user)
```

Comment:
Metrics/LineLength: Line is too long. [103/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)